scipy/stats/_crosstab.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+
+
+def crosstab(*args, levels=None):
+    """
+    Create a table of the counts of the tuples in ``zip(*args)``.
```

Comment:
Is there a more friendly way of phrasing this?   I expect most people looking for this function will be thinking of a 2 way contigency table.  How about

```
Return table giving counts for each possible unique combination of input ``*args``
```

?  As a taster?

Reply:
I rephrased the first line similar to what you suggested.  I tweaked it a bit to keep it under 80 characters.